odoo/tools/convert.py

```diff
@@ -577,14 +578,22 @@ def _tag_record(self, rec):
                         f_val = float(f_val)
                     elif field_type == 'boolean' and isinstance(f_val, str):
                         f_val = str2bool(f_val)
+                    elif field_type in ('one2many', 'many2many'):
+                        if isinstance(f_val, str):
+                            f_val = None
+                        for child in field.findall('./record'):
+                            delay_list.append((child, model._fields[f_name].inverse_name))
```

Comment:
This cannot work for many2many fields, as `inverse_name` is specific to one2many fields.
Let us stick to one2many fields for now, as many2many fields would need a different implementation IMO.